### PR TITLE
Add notebook example for using Dask EntitySets

### LINKED ISSUE #16: Update notebook to use Featuretools Dask Implementation
After https://github.com/FeatureLabs/featuretools/pull/783 is merged, we can update the notebook to the improved api

### LINKED ISSUE #19: AssertionError: target columns not found
After features are generated, there's AssertionError: target columns not found

### LINKED ISSUE #20: Broken link to data
In the read, the link at "here" is broken

> You can download the data directly from Instacart here.

### PR DESCRIPTION
**Pull request by thehomebrewnerd**
Adds a new notebook that demonstrates how to compute a feature matrix using Dask EntitySets. 

Fixes #16 

Fixes #19 

Fixes #20

### CODE REVIEW

**Comment by thehomebrewnerd** [comment]
@rwedge I have removed the original notebook and replaced it with a Dask-only version. Let me know what you think of this setup when you have a chance to review.

**Comment by rwedge** [comment]
> @rwedge I have removed the original notebook and replaced it with a Dask-only version. Let me know what you think of this setup when you have a chance to review.

Some sections of the readme that dealt with there being two approaches should be updated

**Comment by thehomebrewnerd** [comment]
> > @rwedge I have removed the original notebook and replaced it with a Dask-only version. Let me know what you think of this setup when you have a chance to review.
> 
> Some sections of the readme that dealt with there being two approaches should be updated

Updated to remove reference to multiple notebooks.

File: Tutorial.ipynb [hunk 1]
```diff
@@ -213,55 +164,101 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/nate.parsons/dev/featuretools/env/lib/python3.7/site-packages/ipykernel_launcher.py:1: UserWarning: `meta` is not specified, inferred from partial data. Please provide `meta` if the result is unexpected.\n",
```
Comment (rwedge) [inline Tutorial.ipynb:204]: Is there something we can do to resolve this warning?

Comment (thehomebrewnerd) [reply Tutorial.ipynb:204]: I'll have look into the exact cause of that a little more, but I think it might be something we have fix in Featuretools.

Comment (thehomebrewnerd) [reply Tutorial.ipynb:204]: Disregard my previous comment - I didn't look closely enough and thought that warning was happening somewhere else in the notebook. Yes, I think we can fix this warning.

Comment (thehomebrewnerd) [reply Tutorial.ipynb]: @rwedge Added `meta` to the `apply` to fix the previous warning. Notebook should run without any warnings now.

Comment (thehomebrewnerd) [reply Tutorial.ipynb]: @rwedge I think everything has been resolved now - are we ready to merge?

**Comment by thehomebrewnerd** [comment]
@rwedge I want to take another look at this before we merge it in. In particular, there is this code block at the start of the machine learning step:
```
X = fm_encoded.merge(label_times)
X.drop(["user_id", "time"], axis=1, inplace=True)
X = X.fillna(0)
y = X.pop("bought_product").astype('bool')
```
I'm not sure now why I was merging in the label times df, as the labels are already present in the feature matrix, and then we just drop the user_id and time columns anyway, so that merge isn't doing anything for us.

**Comment by thehomebrewnerd** [comment]
> @rwedge I want to take another look at this before we merge it in. In particular, there is this code block at the start of the machine learning step:
> 
> ```
> X = fm_encoded.merge(label_times)
> X.drop(["user_id", "time"], axis=1, inplace=True)
> X = X.fillna(0)
> y = X.pop("bought_product").astype('bool')
> ```
> 
> I'm not sure now why I was merging in the label times df, as the labels are already present in the feature matrix, and then we just drop the user_id and time columns anyway, so that merge isn't doing anything for us.

@rwedge While I was looking into this merge statement, I found another issue that I'm working on resolving. 

First, the merge was not necessary, so that is being removed. The second issue was that when writing the initial feature matrix to disk and then reading it back it, some of the column datatypes were being changed which impacted the encoding and machine learning process. I am updating to skip this intermediate step of writing to disk.

I've made these changes and am now rerunning the notebook to confirm everything works as expected. I hope to have and updated version up for review sometime today.

File: README.md [hunk 2]
```diff
@@ -31,18 +31,15 @@ In this tutorial, we show how [Featuretools](https://www.featuretools.com) can b
...
     
     *You will also need to install **graphviz** for this demo. Please install graphviz according to the instructions in the [Featuretools Documentation](https://docs.featuretools.com/)*
-    
+
+    *Note: In order to run the tutorial, you will need to use Featuretools version 0.16.0 or newer.*
+
+
 3. Download the data
 
     You can download the data directly from Instacart [here](https://www.instacart.com/datasets/grocery-shopping-2017).
```
Comment (rwedge) [inline README.md:40]: We should update this link as well

Comment (thehomebrewnerd) [reply README.md]: @rwedge Good catch. Updated.

**Review by rwedge — APPROVED** [review]
Looks good!

**Pull request merged**